homeassistant/components/sensor/solaredge.py

```diff
@@ -0,0 +1,176 @@
+"""
+Support for SolarEdge Monitoring API.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.solaredge/
+"""
+
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.light import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_API_KEY, CONF_MONITORED_VARIABLES, CONF_NAME)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['solaredge==0.0.2']
+
+DOMAIN = "solaredge"
```

Comment:
Please remove this constant. This platform belongs to the `'sensor'` domain.

Reply:
Hi @MartinHjelmare thanks for the review, all issues should be fixed.